### ISSUE TITLE
Readme out of date?

### ISSUE THREAD

**Issue by iammenasco** [PAK90/mtg-hunter#6]
Am I missing something from the readme? After installation, I am getting a few `Module not found: Error: Cannot resolve module ...` errors. I can add more info if needed, but I feel like I am just missing some little step somewhere.

**Comment by iammenasco** [comment]
@PAK90 - Thoughts?

**Comment by PAK90** [comment]
Whoops, sorry, missed this alert the first time. Umm... I did a fresh install on an ubuntu 14.04 system and I didn't get that error; which modules does it say it's missing, and what OS are you using?

**Comment by iammenasco** [comment]
```
ERROR in ./src/TogglePanel.js
Module not found: Error: Cannot resolve module 'bem-cn' in [redacted]/mtg-hunter/src
 @ ./src/TogglePanel.js 37:15-32

ERROR in ./src/Tabs.js
Module not found: Error: Cannot resolve module 'js-stylesheet' in [redacted]/mtg-hunter/src
 @ ./src/Tabs.js 15:20-44

ERROR in ./~/css-loader?sourceMap!./~/sass-loader?sourceMap!./~/toolbox-loader!./src/styles/customisations.scss
Module not found: Error: Cannot resolve module 'rc-slider/assets/index.css' in [redacted]/mtg-hunter/src/styles
 @ ./~/css-loader?sourceMap!./~/sass-loader?sourceMap!./~/toolbox-loader!./src/styles/customisations.scss 5:10-100

ERROR in ./src/styles/customisations.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Cannot resolve module 'rc-slider/assets/index.css' in [redacted]/mtg-hunter/src/styles
    at [redacted]/mtg-hunter/node_modules/webpack/lib/Compilation.js:229:38
    at onDoneResolving ([redacted]/mtg-hunter/node_modules/webpack/lib/NormalModuleFactory.js:29:20)
    at [redacted]/mtg-hunter/node_modules/webpack/lib/NormalModuleFactory.js:85:20
    at [redacted]/mtg-hunter/node_modules/webpack/node_modules/async/lib/async.js:726:13
    at [redacted]/mtg-hunter/node_modules/webpack/node_modules/async/lib/async.js:52:16
    at done ([redacted]/mtg-hunter/node_modules/webpack/node_modules/async/lib/async.js:241:17)
    at [redacted]/mtg-hunter/node_modules/webpack/node_modules/async/lib/async.js:44:16
    at [redacted]/mtg-hunter/node_modules/webpack/node_modules/async/lib/async.js:723:17
    at [redacted]/mtg-hunter/node_modules/webpack/node_modules/async/lib/async.js:167:37
    at [redacted]/mtg-hunter/node_modules/webpack/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:24:19
```

Note - Running Mac OS 10.11.5
Other stats:

```
{ hunter: '1.0.0',
  npm: '2.14.12',
  ares: '1.10.1-DEV',
  http_parser: '2.5.0',
  icu: '56.1',
  modules: '46',
  node: '4.2.5',
  openssl: '1.0.2e',
  uv: '1.8.0',
  v8: '4.5.103.35',
  zlib: '1.2.8' }
```

**Comment by PAK90** [comment]
At first glance I don't see anything obvious. I will say I've never tried running my project on a Mac OS, so that could be it... though it shouldn't be that, given how it works on other Linux-based systems.

**Comment by iammenasco** [comment]
What node version are you running?

**Comment by PAK90** [comment]
v5.4.0.

**Comment by iammenasco** [comment]
Ok, I got it up and running. I have a few things I am looking into, and I'll send a pull request once I have it fully working for you to look at.

**Comment by PAK90** [comment]
Cool :) What was the fix? And what are you looking into, out of curiosity?

**Comment by iammenasco** [comment]
The fix is here: #7 

Here are two other issues I've noticed. I'll look into it later tonight or tomorrow, hopefully.

### `404`s on a bunch of local images

#### Local

![screen shot 2016-05-25 at 9 57 59 am](https://cloud.githubusercontent.com/assets/5480206/15547027/5bc23138-225f-11e6-895d-e3ff895bfd23.png)

#### [Production](http://mtg-hunter.com)

<img width="1004" alt="screen shot 2016-05-25 at 9 58 10 am" src="https://cloud.githubusercontent.com/assets/5480206/15547051/6e630ab0-225f-11e6-9b55-998d44436598.png">

### features are missing (card prices?):

#### Local

![screen shot 2016-05-25 at 9 57 30 am](https://cloud.githubusercontent.com/assets/5480206/15547105/a36b759e-225f-11e6-9de2-bd05bb83f759.png)

#### [Production](http://mtg-hunter.com)

<img width="280" alt="screen shot 2016-05-25 at 9 57 37 am" src="https://cloud.githubusercontent.com/assets/5480206/15547124/b552e8a0-225f-11e6-901d-354ef22075f4.png">

**Comment by PAK90** [comment]
Ah yeah. I don't host the mana symbols and set symbols on github because that's a surefire way of getting a takedown request. And prices are missing because they're an additional piece of information that's sent to the elasticsearch server by the getPrices.js script (since it's run every 24 hours to keep them current); they're not included in the initial processing of the mtgjson data.

**Comment by iammenasco** [comment]
Interesting. That helps alot. I see the script for the prices, I'll run it. And how could I get my hand on those fancy icons?

I think some assets are allowed on github, as long as its not a crazy amount:

> We recommend repositories be kept under 1GB each. This limit is easy to stay within if large files are kept out of the repository. If your repository exceeds 1GB, you might receive a polite email from GitHub Support requesting that you reduce the size of the repository to bring it back down.

**Comment by PAK90** [comment]
Before you run the price script, comment out line 11 and uncomment line 12.
The assets thing is more WotC not wanting their symbols in a publicly-accessible place available for easy download.

**Comment by iammenasco** [comment]
That makes sense, thanks! I'll see if I can get them elsewhere then.

**Comment by PAK90** [comment]
Use this for now; https://github.com/PAK90/mtg-hunter/tree/85f130e602bbfb4c9c498363158479f4e9f42800/src/img
Let me know when you have all the images.

**Comment by iammenasco** [comment]
Got it, thanks a ton!

**Comment by iammenasco** [comment]
Images and prices are working great now. The main installation issue should now be resolved with the PR #7. I've added some notes to the `README.md` for getting prices manually. I can add that to the PR if you'd like.

**Comment by iammenasco** [comment]
This issue should now be closed, since it has been resolved with pull request #7.

I've added pull request #8 to update the read me individually.
